airflow/www/app.py

```diff
@@ -70,8 +71,27 @@ def create_app(config=None, testing=False, app_name="Airflow"):
     flask_app = Flask(__name__)
     flask_app.secret_key = conf.get('webserver', 'SECRET_KEY')
 
-    session_lifetime_days = conf.getint('webserver', 'SESSION_LIFETIME_DAYS', fallback=30)
-    flask_app.config['PERMANENT_SESSION_LIFETIME'] = timedelta(days=session_lifetime_days)
+    default_session_lifetime = 43200
```

Comment:
A minor one: I suggest to make the name more explicit, say `default_session_lifetime_mins`.